CHANGELOG.md

```diff
@@ -36,6 +36,9 @@ This project adheres to [Semantic Versioning](https://semver.org/).
   ([#880](https://github.com/MyIntervals/emogrifier/pull/880))
 
 ### Fixed
+- Fix a nonsensical code example in the README
+  ([#920](https://github.com/MyIntervals/emogrifier/pull/920),
```

Comment:
URL should contain `issues` rather than `pull` (if we want to reference the issue as well as the PR).

Reply:
Fixed, rebased and repushed.